jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/ssl/SslContextFactory.java

```diff
@@ -1615,7 +1612,9 @@ public void setHostnameVerifier(HostnameVerifier hostnameVerifier)
      *
      * @param realm the realm
      * @return the Password object
+     * @deprecated use {#link getCredential} instead.
      */
+    @Deprecated(since = "12.0.13", forRemoval = true)
```

Comment:
This method is now not called by the implementation, and could break those that have overridden it.

Unfortunately, I see no way to call it to maintain backwards compatibility... @gregw wdyt?
Should we move this PR to 12.1.x?